docs/content/Examples-Tutorials-Recipes/Pagination.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Pagination
+permalink: /recipes/pagination
+category: Examples & Tutorials
+subCategory: Queries
+menuOrder: 2
+---
+
+## Use case
+
+We want to display a table of data with hundreds of rows. For the convenience of
+displaying, we will use a pagination. At the recipe below, we will get the
+orders list sorted by the order number. Every page will have 5 orders.
+
+## Query
+
+To creating the pagination we can use the `limit` and `offset` query properties.
+First, we get the number of all orders that we have. Then we should set the
+`limit` and `offset` properties for the queries that will get the orders from
+Cube.js API.
+
+```bash
+// Get count of the orders
+curl cube:4000/cubejs-api/v1/load \
+-G -s --data-urlencode "query={"measures": ["Orders.count"], "timeDimensions": [{"dimension": "Orders.createdAt"}]}"
+```
+
+```bash
+// Get first five orders
+curl cube:4000/cubejs-api/v1/load \
+-G -s --data-urlencode "query={"order": [["Orders.number", "asc"]], "dimensions": ["Orders.number"], "limit": 5}"
+```
+
+```bash
+// Get next five orders
+curl cube:4000/cubejs-api/v1/load \
+-G -s --data-urlencode "query={"order": [["Orders.number", "asc"]], "dimensions": ["Orders.number"], "limit": 5, "offset": 5}"
+```
+
+## Result
+
+We have received five orders per query and can use it as we want.
+
+```javascript
```

Comment:
```suggestion
```json
```

Reply:
@hassankhan Do you like how JSON is highlighted better than JavaScript? 😄